src/cpptests/redismock/redismock.cpp

```diff
@@ -546,7 +554,15 @@ void KVDB::debugDump() const {
 std::map<std::string, void *> fnregistry;
 #define REGISTER_API(basename) fnregistry["RedisModule_" #basename] = (void *)RMCK_##basename
 
+extern "C" {
+int moduleRegisterApi(const char *funcname, void *funcptr) {
+  fnregistry[funcname] = funcptr;
+  return 0;
+}
+}
+
 static void registerApis() {
+  REGISTER_API(GetApi);
```

Comment:
This is only if there is a `RedisModule_StringCompare`

Reply:
there is!